src/tests.rs

```diff
@@ -534,7 +534,51 @@ fn check_elo_stronger_looses() {
 }
 
 #[test]
-fn check_elo_player_aborts() {
+fn check_elo_player_draw() {
+    new_test_ext().execute_with(|| {
+        System::set_block_number(1);
+        let charlie: u64 = account("Charlie", 0, 2);
+        let dave:u64  = account("Dave", 0, 3);
+
+        let bet_asset_id = AssetId::get();
+        let bet_amount = AssetMinBalance::get() * 5;
+
+        assert_ok!(Chess::create_match(
+            RuntimeOrigin::signed(charlie),
+            dave,
+            MatchStyle::Bullet,
+            bet_asset_id,
+            bet_amount
+        ));
+
+        let match_id = Chess::chess_match_id_from_nonce(0).unwrap();
+
+        assert_ok!(Chess::join_match(RuntimeOrigin::signed(dave), match_id));
+
+        // check elo before the match finishes
+        assert_eq!(Chess::player_elo(charlie), 1000);
+        assert_eq!(Chess::player_elo(dave), 1000);
```

Comment:
If both players have the same ELO and make draws then they maintain the same ELO, that's correct. However, in the previous example it what checked what happens if they have different ELO, which I think it's interesting to maintain.